templates/new-task.html

```diff
@@ -1,55 +1,44 @@
 {% extends "layout.html" %}
 {% block content %}
     <form action="" method=post>
-    {% if todo is defined %}
-        <h2>Edit TODO</h2>
-    {% else %}
-        <h2>Create new TODO</h2>
-    {% endif %}
-
+    <h2>{%- if todo is defined %} Edit {% else %} Create new {% endif -%}Todo</h2>
         <div class="control-group">
-
-
             <div class="control-group" >
                 <div class="controls">
                     <select class="selectpicker" name="category" required>
                         {% for category in categories %}
-                            {% if todo is defined %}
-                            <option value="{{ category.id }}" {% if todo.category_id == category.id %} selected {% endif %}>{{ category.name }}</option>
-                            {% else %}
-                            <option value="{{ category.id }}">{{ category.name }}</option>
-                            {% endif %}
+                            <option value="{{ category.id }}" {%if todo is defined %}{% if todo.category_id == category.id %} selected {% endif %}{% endif %}>{{ category.name }}</option>
                         {% endfor %}
-
                     </select>
                 </div>
             </div>
+            {#
```

Comment:
i didnt actually remove anything related to priority, just commented it out in places, like here